crates/egui/src/context.rs

```diff
@@ -3487,6 +3543,22 @@ impl Context {
         })
     }
 
+    #[cfg(feature = "accesskit")]
+    pub(crate) fn register_accesskit_parent(&self, id: Id, parent_id: Id) {
+        self.write(|ctx| {
+            if let Some(state) = ctx.viewport().this_pass.accesskit_state.as_mut() {
+                state.parent_map.insert(id, parent_id);
+                state.child_map.entry(id).or_default().parent = Some(parent_id);
```

Comment:
Why do we need both of these? Looks like just duplicated state?

Isn't `parent_map[foo] == state.child_map[foo].parent` always ?